react/ContactPicker/Readme.md

```diff
@@ -1,16 +1,22 @@
 ```jsx
-import ContactPicker from 'cozy-ui/transpiled/react/ContactPicker';
-import DemoProvider from '../ContactsListModal/DemoProvider';
-initialState = { selectedContact: null };
+import { BreakpointsProvider } from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import ContactPicker from 'cozy-ui/transpiled/react/ContactPicker'
+import DemoProvider from '../ContactsListModal/DemoProvider'
 
-<DemoProvider>
-  <ContactPicker
-    placeholder="Select a contact"
-    onChange={selectedContact => setState({ selectedContact })}
-    value={state.selectedContact}
-    listPlaceholder="Search a contact"
-    listEmptyMessage="No contact"
-    addContactLabel="Add a contact"
-  />
-</DemoProvider>
+initialState = { selectedContact: null }
+
+;
```

Comment:
What's the point of this semi colon?

Reply:
pour séparer la partie markdown du code react et ainsi interpréter le code react